topics/statistics/tutorials/CNN/workflows/Intro_To_CNN_v1_0_11_0-test.yml

```diff
@@ -0,0 +1,23 @@
+- doc: Test outline for Intro_To_CNN_v1_0_11_0
```

Comment:
I am not sure how to test this automatic workflow testing

Reply:
https://planemo.readthedocs.io/en/latest/running.html#the-basics

planemo test workflows.ga :)

Reply:
you don't need to run the tests yourself, that's the GTN's responsibility. We're not doing it currently, but we will at some point in the future, automatically, so, as long as a test is present, it's fine to merge (assuming the rest looks good + all GTN tests are passing.)